### PR TITLE
Update deps and add f-e-d-c

### PR DESCRIPTION
**Pull request by vulongm**
For Krita f-e-d-c would need to wait for the AppData patches to make it to a release, made a PR with the patches used. https://invent.kde.org/graphics/krita/-/merge_requests/1463

### CODE REVIEW

File: org.kde.krita.yaml [hunk 1]
```diff
@@ -14,7 +14,6 @@ finish-args:
   - --socket=pulseaudio
   - --device=dri
   - --filesystem=host
-  - --filesystem=xdg-config/kdeglobals:ro
```
Comment (travier) [inline org.kde.krita.yaml:17]: Do you know shy it was needed before and not anymore?

File: org.kde.krita.yaml [hunk 2]
```diff
@@ -492,22 +521,29 @@ modules:
...
+          type: anitya
+          project-id: 1709
+          stable-only: true
+          url-template: https://www.libraw.org/data/LibRaw-$version.tar.gz
       - type: patch
         paths:
           - patches/libraw/0001-LibRaw-cmake.patch
           - patches/libraw/0002-Use-CMake-install-prefix.patch
   - name: gsl
-    buildsystem: cmake-ninja
```
Comment (travier) [inline org.kde.krita.yaml:500]: Looks weird. Are they no longer using cmake?

**Review by travier — COMMENTED** [review]
Looks good. Some minor comments.

File: org.kde.krita.yaml (hunk 1 above)
Comment (vulongm) [reply org.kde.krita.yaml:17]: The file contains information for KDE Plasma's system theme like Breeze and the colors. But the Krita flatpak uses it's own styling with Fusion. If we are able to switch to the KDE Runtime in the future as well it is specified there as well. https://invent.kde.org/packaging/flatpak-kde-runtime/-/blob/qt5.15lts-21.08/org.kde.Sdk.json.in#L78

File: org.kde.krita.yaml (hunk 2 above)
Comment (vulongm) [reply org.kde.krita.yaml:500]: The version used before at https://github.com/ampl/gsl/ includes the CMake build support but that isn't on the release monitor site looking it up so I switched it to the one they had at https://release-monitoring.org/project/1267/.

**Comment by travier** [comment]
OK, let's give it a try.

**Pull request merged**